src/Workspaces/CSharp/Portable/Extensions/ITypeSymbolExtensions.TypeSyntaxGeneratorVisitor.cs

```diff
@@ -241,10 +247,20 @@ public override TypeSyntax VisitNamespace(INamespaceSymbol symbol)
                 }
             }
 
+            private TypeSyntax AddGlobalAlias(INamespaceOrTypeSymbol symbol, SimpleNameSyntax syntax)
+            {
+                return AddInformationTo(
+                    SyntaxFactory.AliasQualifiedName(
+                        CreateGlobalIdentifier(),
```

Comment:
What happens if the type in question is being referenced through an extern such that this qualification won't work?